public/tailwindPlugin.js

```diff
@@ -45,13 +50,33 @@ const fontFamily = {
   mono: ['InconsolataVariable', ...defaultTheme.fontFamily.mono],
 }
 
+const keyframes = {
+  'accordion-down': {
+    from: { height: 0 },
+    to: { height: 'var(--radix-accordion-content-height)' },
+  },
+  'accordion-up': {
+    from: { height: 'var(--radix-accordion-content-height)' },
+    to: { height: 0 },
+  },
```

Comment:
Is this best-practice? I'd expect since these are inverse animations to be able to reverse the animation direction and get the same effect with fewer lines of code